circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
```

Comment:
For posterity: https://gist.github.com/mattbennett/aa7cf713c97e689c56257991231471e5